src/components/Project/ProjectDashboard.tsx

```diff
@@ -118,50 +131,93 @@ export const ProjectExecutions: React.FC<ProjectExecutionsProps> = ({
     },
   );
 
-  const fetch = React.useCallback(() => query.fetchNextPage(), [query]);
+  const fetch = React.useCallback(() => executionsQuery.fetchNextPage(), [executionsQuery]);
+
+  const { value: workflows } = useWorkflowNameList({ domain, project }, {});
+  const numberOfWorkflows = workflows.length;
+  const { value: tasks } = useTaskNameList({ domain, project }, {});
+  const numberOfTasks = tasks.length;
+
+  const queryClient = useQueryClient();
 
-  const content = query.isLoadingError ? (
-    <DataError error={query.error} errorTitle={failedToLoadExecutionsString} retry={fetch} />
-  ) : query.isLoading ? (
+  const projectDomainAttributesQuery = useQuery<Admin.ProjectDomainAttributesGetResponse, Error>({
+    queryKey: ['projectDomainAttributes', project, domain],
+    queryFn: async () => {
+      const projectDomainAtributes = await getProjectDomainAttributes({ domain, project });
+      queryClient.setQueryData(
+        ['projectDomainAttributes', project, domain],
+        projectDomainAtributes,
+      );
+      return projectDomainAtributes;
+    },
+    staleTime: Infinity,
+  });
+
+  const content = executionsQuery.isLoadingError ? (
+    <DataError
+      error={executionsQuery.error}
+      errorTitle={failedToLoadExecutionsString}
+      retry={fetch}
+    />
+  ) : executionsQuery.isLoading ? (
     <LargeLoadingSpinner />
   ) : (
     <WorkflowExecutionsTable
       key={tableKey}
       fetch={fetch}
       value={executions}
-      lastError={query.error}
-      moreItemsAvailable={!!query.hasNextPage}
+      lastError={executionsQuery.error}
+      moreItemsAvailable={!!executionsQuery.hasNextPage}
       showWorkflowName={true}
-      isFetching={query.isFetching}
+      isFetching={executionsQuery.isFetching}
       data-testid="workflow-table"
     />
   );
 
+  const configData =
+    projectDomainAttributesQuery.data?.attributes?.matchingAttributes?.workflowExecutionConfig ??
+    undefined;
+
+  const renderDomainSettingsSection = () => (
+    <div>
+      <div className={styles.projectStats}>
+        <Typography variant="h5">{t('workflowsTotal', numberOfWorkflows)}</Typography>
+        <Typography variant="h5">{t('tasksTotal', numberOfTasks)}</Typography>
+      </div>
```

Comment:
I think this section with number of workflows and tasks should be always there. Otherwise it gives no additional information comparing to ExecutionPage when DomainSettings is not defined.

Reply:
reverted this change!